SparkPost/2web2ui
# FE-766 - Remove in-app notifications

FE-766 - Remove in-app notifications

### What Changed
 - Removed dropdown from top navigation
 - Removed all markdown files and code to import them including webpack configuration

### How To Test
 - verify there is no in app notification
 - verify the storybook still works
 - verify that all components that were used only by in-app notification are removed.

### To Do
